workspaces/marketplace/plugins/marketplace-backend/src/router.ts

```diff
@@ -108,6 +159,57 @@ export async function createRouter({
     res.json(plugin);
   });
 
+  router.get('/plugin/:namespace/:name/configuration', async (req, res) => {
+    const [readDecision, installDecision] = await Promise.all([
+      authorizeConditional(req, extensionPluginReadPermission),
+      authorizeConditional(req, extensionPluginCreatePermission),
```

Comment:
I wonder if this should just be the read permission only and then we can use the create permission for `post`.